src/sections/dataset/dataset-metadata/DatasetMetadata.tsx

```diff
@@ -0,0 +1,21 @@
+import { Accordion } from 'dataverse-design-system'
+import { DatasetMetadataBlock as DatasetMetadataBlockModel } from '../../../dataset/domain/models/Dataset'
+import { DatasetMetadataBlock } from './dataset-metadata-block/DatasetMetadataBlock'
+
+interface DatasetMetadataProps {
+  metadataBlocks: DatasetMetadataBlockModel[]
+}
+
+export function DatasetMetadata({ metadataBlocks }: DatasetMetadataProps) {
+  const allKeys = metadataBlocks.map((_, index) => index.toString())
+
+  return (
+    <Accordion defaultActiveKey={allKeys} alwaysOpen>
```

Comment:
In the JSF Dataverse UI, the Accordion only shows the first block open, the rest are closed

Reply:
If we change the key to ['0'], it will match the JSF version

Reply:
Done! Thanks!